ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml

```diff
@@ -55,20 +55,6 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
-- as: aws-ipi-disc-priv-sts-ep-f60
-  cron: 47 8 22 1,3,5,7,9,11 *
-  steps:
-    cluster_profile: aws-qe
-    dependencies:
-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: m6g.xlarge
-      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
-      OCP_ARCH: arm64
-    test:
-    - chain: cucushift-installer-check-cluster-health
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
 - as: aws-ipi-proxy-sts-f60
```

Comment:
@jinyunma do we also need drop this one, this is actually being covered in the amd64 as well - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml#L81

Reply:
I only updated jobs where the install workflow contains step `enable-qe-catalogsource-disconnected`, didn't touch others.
But I can update more jobs in arm64 nightly and arm64 stable based on latest discussion.

Reply:
@gpei updated.